index.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = YahooFantasy;
 
-var OAuth = require('oauth').OAuth,
-  https = require('https'),
+var request = require('request'),
```

Comment:
With OAuth 2.0, we must use API requests which be made over HTTPS and is contained with the access token. ([link](https://developer.yahoo.com/oauth2/guide/apirequests/)) If you want to query the public data(ex. game.meta) with Client credentials(Consumer Key / Consumer Secret), you might mix to use OAuth 2.0 and OAuth 1.0 in yfsapi. :disappointed_relieved: